types/mongodb/index.d.ts

```diff
@@ -230,11 +230,11 @@ export type WithTransactionCallback<T> = (session: ClientSession) => Promise<T>;
  * see {@link http://mongodb.github.io/node-mongodb-native/3.5/api/MongoError.html}
  */
 export class MongoError extends Error {
-    constructor(message: string);
+    constructor(message: string | object);
```

Comment:
It seems to accept also an `Error`, maybe we could add it to the types?

Reply:
To me, an instance of `Error` in an `object`.
[TypeScript documentation](https://www.typescriptlang.org/docs/handbook/basic-types.html#object):
> object is a type that represents the non-primitive type, i.e. anything that is not number, string, boolean, bigint, symbol, null, or undefined.

Reply:
Yes, but it's more readable by an user if also Error is listed. I don't think typescript will omit it from the intellisense even if its technically superseded by object

Reply:
Done :)